src/android/dialonce.gradle

```diff
@@ -3,5 +3,7 @@ repositories {
 }
 
 dependencies {
-    compile 'com.dialonce:dialonce-android:+'
+    compile ('com.dialonce:dialonce-android:+') {
+    	exclude group: 'com.google.code.findbugs'
```

Comment:
@CAMOBAP795 What will be the behaviour if we exclude this and the app doesn't have this dependency installed via another plugin ?

Reply:
`exclude` only affect dependency resolving step, it means break transitive dependency.

So if there no another  `'com.google.code.findbugs'` it will be still downloaded

Reply:
@jkernech let me know if you have more questions/feedback

Reply:
👍